py3status/modules/mpris.py

```diff
@@ -0,0 +1,382 @@
+    format_stream: see placeholders below, keep in mind that {artist} and
+            {album} are empty. This format is also used for videos or just media
+            files without an artist information or any other metadata!
+    format_none: define output if no player is running
+    icon_pause: text for the pause button in the button control panel
+    icon_play: text for the play button in the button control panel
+    icon_stop: text for the stop button in the button control panel
+    icon_next: text for the next button in the button control panel
+    icon_previous: text for the previous button in the button control panel
+    state_pause: text for placeholder {state} when song is paused
+    state_play: text for placeholder {state} when song is playing
+    state_stop: text for placeholder {state} when song is stopped
+    player_priority: priority of the players.
+            Keep in mind that the state has a higher priority than
+            player_priority. So when player_priority is "mpd,bomi" and mpd is
+            paused and bomi is playing than bomi wins.
+
+Format of status string placeholders:
+    {album} album name
+    {artist} artiste name (first one)
+    {length} time duration of the song
+    {player} show name of the player
+    {state} playback status of the player
+    {time} played time of the song
+    {title} name of the song
+
+i3status.conf example:
+
+```
+mpris {
+    format = "{player}: {state} {artist} - {title} [{time} / {length}]"
+    format_stream = "{player}: {state} {title} [{time} / {length}]"
+    format_none = "no player"
+    buttons_control = "left"
+    buttons_order = "previous,play,next"
+    player_priority = "mpd,cantata,vlc,bomi,*"
+}
+```
+
+only show information from vlc:
+
+```
+mpris {
+    player_priority = "vlc"
+}
+```
+
+only show information from mpd and vlc, but mpd has a higher priority:
+
+```
+mpris {
+    player_priority = "mpd,vlc"
+}
+```
+
+show information of all players, but mpd and vlc have the highest priority:
+
+```
+mpris {
+    player_priority = "mpd,vlc,*"
+}
+```
+
+Requires:
+    pydbus
+
+@author Pierre Guilbert, Jimmy Garpehäll, sondrele, Andrwe, Moritz Lüdecke
+"""
+
+from datetime import timedelta
+from time import time
+import re
+from pydbus import SessionBus
+
+
+SERVICE_BUS = 'org.mpris.MediaPlayer2'
+INTERFACE = SERVICE_BUS + '.Player'
+SERVICE_BUS_URL = '/org/mpris/MediaPlayer2'
+SERVICE_BUS_REGEX = '^' + re.sub(r'\.', '\.', SERVICE_BUS) + '.'
+UNKNOWN = 'Unknown'
+UNKNOWN_TIME = '-:--'
+
+
+def _get_time_str(microseconds):
+    seconds = int(microseconds / 1000000)
+    time = str(timedelta(seconds=seconds))
+    if time[0] == '0':
+        time = time[2:]
+        if time[0] == '0':
+            time = time[1:]
+
+    return time
+
+
+class Py3status:
+    """
+    """
+    # available configuration parameters
+    button_stop = None
+    button_toggle = 1
+    button_next = 4
+    button_previous = 5
+    buttons_control = None
```

Comment:
I'm not sure of this name it is a bit too close to `button_xxx` maybe something like `show_controls`?